src/components/NavigationBar/index.jsx

```diff
@@ -41,115 +47,242 @@ function NavigationBar({ theme }) {
-            }}
-            variant={'contained'}
+                <MenuItem value={'pt-PT'}>🇵🇹 PT</MenuItem>
+                <MenuItem value={'en-GB'}>🇬🇧 EN</MenuItem>
+              </Select>
+
+              <IconButton
+                color={'inherit'}
+                onClick={toggleDarkMode}
+                size={'small'}
+                sx={{ '&:hover': { backgroundColor: '#000000' }, backgroundColor: '#495057' }}
+              >
+                {darkMode ? (
+                  <Brightness7
+                    fontSize={'inherit'}
+                    sx={{ color: 'common.white' }}
+                  />
+                ) : (
+                  <Brightness4
+                    fontSize={'inherit'}
+                    sx={{ color: 'common.white' }}
+                  />
+                )}
+              </IconButton>
+            </Hidden>
+
+            <Hidden mdUp>
+              <IconButton
+                aria-label={'menu'}
+                color={'inherit'}
+                edge={'end'}
+                onClick={handleDrawerOpen}
+              >
+                <MenuIcon />
+              </IconButton>
+            </Hidden>
+          </Toolbar>
+        </Container>
+      </AppBar>
+      <Drawer
+        anchor={'right'}
+        onClose={handleDrawerClose}
+        open={drawerOpen}
+      >
+        <List sx={{ width: 250 }}>
+          <ListItem
+            button
+            component={Link}
+            to={'/'}
+          >
+            <ListItemText primary={t('menu.home')} />
+          </ListItem>
+          <ListItem
+            button
+            component={Link}
+            to={'/product/list'}
           >
-            {darkMode ? (
-              <Brightness7
-                fontSize={'inherit'}
-                style={{ color: 'white' }}
-                // eslint-disable-next-line react/jsx-closing-bracket-location
-              />
-            ) : (
-              <Brightness4
-                fontSize={'inherit'}
-                style={{ color: 'white' }}
-                // eslint-disable-next-line react/jsx-closing-bracket-location
-              />
-            )}
-          </IconButton>
-        </Navbar.Collapse>
-      </Container>
-    </Navbar>
+            <ListItemText
+              primary={t('menu.product-list')}
+              secondary={
+                <Badge
+                  badgeContent={numberOfProducts()}
+                  color={'error'}
+                />
+              }
+            />
+          </ListItem>
+          {mobileAppUrl && (
+            <ListItem
+              button
+              component={'a'}
+              href={mobileAppUrl}
+              rel={'noopener noreferrer'}
+              target={'_blank'}
+            >
+              <ListItemText primary={t('menu.mobile-app')} />
+            </ListItem>
+          )}
+          <Divider />
+          <ListItem disableGutters>
+            <ListItemText primary={t('menu.language')} />
```

Comment:
add padding inline of 16 px to be consistent with other elements (mobile)

Reply:
this is still an ongoing issue on mobile

Reply:
there is also a dividir in between the list and language element - what is it doing?